dummyfs/dummyfs.c

```diff
@@ -1010,10 +1177,11 @@ int dummyfs_mount(void **ctx, const char *data, unsigned long mode, oid_t *root)
 
 int dummyfs_unmount(void *ctx)
 {
+	TRACE;
 	dummyfs_t *fs = (dummyfs_t *)ctx;
 
 	/* Assume, no other thread uses this filesystem */
```

Comment:
how safe is this assumption?

Reply:
Pretty safe, there's little point to use dummyfs multithreaded.
Also, we need to destroy mutex anyway